{{cookiecutter.project_name}}/notebooks/ModelValidation.py

```diff
@@ -37,8 +37,11 @@
 
 # COMMAND ----------
 
-dbutils.widgets.dropdown("env", "prod", ["staging", "prod"], "Environment Name")
+dbutils.widgets.dropdown("env", "prod", ["staging", "prod"], "Environment Name(decide input data)")
```

Comment:
Yeah this is a tricky rename. Maybe something like this so it doesn't get cut off and the use makes sense.
```suggestion
dbutils.widgets.dropdown("env", "prod", ["staging", "prod"], "Environment (for input data)")
```

Reply:
Sure. cut-off content will be shown in a hover card.